components/crud-web-apps/common/frontend/kubeflow-common-lib/projects/kubeflow/src/lib/services/namespace.service.ts

```diff
@@ -15,9 +15,11 @@ export class NamespaceService {
 
   // Observable string sources
   private selectedNamespaceSource = new ReplaySubject<string>(1);
+  private dashboardConnectedSource = new BehaviorSubject<boolean>(true);
```

Comment:
Hello @kimwnasptd. I think this Observable should not start with `true` as a default value. If someone uses the `dashboardConnected$` property and the `centraldashboard` is not ready, it will wrongly inform the user that it is ready. Maybe you can use `null` as the unknown state or even better use an enum. Something like this:
```
enum DashboardState {
  Unknown = 0, // or Connecting
  Connected,
  Disconnected,
}
```

Reply:
Thank you for the feedback @tasos-ale! 

The reason I used `true` as a default value here is because if it starts as `null` then the app will think that the dashboard is not present and try to make a request to fetch the namespaces. 

So in this case I take for granted the opposite, which is expect the dashboard to be present and if after the check it's missing then the apps will make the request to fetch the namespaces

Reply:
but indeed the enumeration is a better way to express this. I'll use this enum instead of a boolean value and the apps will fetch the namespaces only if the state is `Disconnected`

Reply:
pushed a commit for the described changes